src/__tests__/reporter/startSuiteTestReporting.spec.ts

```diff
@@ -158,4 +159,32 @@ describe('start reporting suite/test', () => {
     expect(reporter.suites).toEqual(new Map());
     expect(reporter.testItems).toEqual(new Map());
   });
+
+  test('@smoke client.startTestItem should be called with one tag at the beginning', () => {
```

Comment:
Did you run the tests locally? I see that they are failing in CI.
Also the way its written won't work as you need to add a `@smoke` to the `simpleTestCase`'s `title` to test the feature.

Reply:
Hi @AmsterGet ,
At present, I'm encountering an issue with the tests locally as they are all failing, including the older ones. As a result, I'm currently validating the tests on GitHub actions.